lib/api/core/validation/types/string.js

```diff
@@ -0,0 +1,61 @@
+var
+  BaseConstructor = require('../baseType'),
+  allowedLengthOptions = ['min', 'max'];
+
+/**
+ * @constructor
+ */
+function StringType () {
+  this.typeName = 'string';
+  this.allowChildren = false;
+  this.allowedTypeOptions = ['length'];
+}
+
+StringType.prototype = new BaseConstructor();
+
+/**
+ * @param {TypeOptions} typeOptions
+ * @param {*} fieldValue
+ * @param {string[]} errorMessages
+ */
+StringType.prototype.validate = function (typeOptions, fieldValue, errorMessages) {
+  if (typeof fieldValue !== 'string') {
+    errorMessages.push('The field must be a string.');
+    return false;
+  }
+
+  if (!typeOptions.length) {
+    return true;
+  }
+
+  if (typeOptions.length.hasOwnProperty('min') && fieldValue.length < typeOptions.length.min) {
+    errorMessages.push('The string is not long enough.');
```

Comment:
Invalid string length. Expected min: ${min} max: ${max}. Received: ${fieldValue.lengh} "${fieldValue}"

Can then also be merged with the over max case below.